### DISCUSSION TITLE
Define an initially small public stable API

### DISCUSSION THREAD

**Discussion by paul-marechal** [eclipse-theia/theia#11125] [Improvements]
We are planning to define an initial small public stable API and grow it with time based on needs/requests.

You can input your feedback about what to include in the beginning here: https://github.com/eclipse-theia/theia/discussions/11124

Implementation-wise I believe we should follow this idea: https://github.com/eclipse-theia/theia/discussions/10772

Thoughts?

**Comment by Niklas-Dahlquist** [comment]
Great initiative looking forward to having some stable APIs, good to take a first step to a more stable release process, cheers
